server/webhook_parser.go

```diff
@@ -144,14 +145,16 @@ func parseWebhookChangeLog(jwh *JiraWebhook) Webhook {
 
 		if event != nil {
 			events = append(events, event)
+		} else {
+			fieldsNotFound = append(fieldsNotFound, field)
 		}
 	}
 	if len(events) == 0 {
-		return nil
+		return nil, fmt.Errorf("Error parsing webhook. Unsupported fields: %s", strings.Join(fieldsNotFound, ", "))
```

Comment:
So we'll only return the error if there were no recognized fields... What about cases where there was one known field and one (or more) unrecognized fields? Should we log something?

Reply:
I agree, it seems awkward to approach it this way. The fact that there are fields that are not supported doesn't seem much like an error to me, but the  error message being returned when the webhook is not posting anything needs the context provided here. I also don't want to spam the logs if there's not actually something wrong. How do you think we should approach this @cpoile?

Reply:
There was some discussion this morning in the Jira plugin channel pertaining to users not getting posts when they expected to get them, which is what brought this issue up.

Reply:
Okay, how about just leave it like you have it -- they'll get a webhook for something at least, which solves the "we didn't get anything". And an error otherwise.